server/src/test/java/org/elasticsearch/reservedstate/service/FileSettingsServiceTests.java

```diff
@@ -355,17 +368,21 @@ public void testHandleSnapshotRestoreResetsMetadata() throws Exception {
     private static void writeTestFile(Path path, String contents) {
         Path tempFile = null;
         try {
+            logger.info("Creating settings temp file under [{}]", path.toAbsolutePath());
             tempFile = Files.createTempFile(path.getParent(), path.getFileName().toString(), "tmp");
+            logger.info("Created settings temp file [{}]", tempFile.getFileName());
             Files.writeString(tempFile, contents);
 
             try {
+                logger.info("Moving settings temp file to replace [{}]", tempFile.getFileName());
                 Files.move(tempFile, path, REPLACE_EXISTING, ATOMIC_MOVE);
             } catch (AtomicMoveNotSupportedException e) {
```

Comment:
Could be interesting to know if we hit this catch.

Reply:
++ added